backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/json/Schema.java

```diff
@@ -25,7 +25,7 @@ public Schema(SchemaMetadata schema, boolean minimal) {
             .map(entry -> new Setting(entry.getKey(), entry.getValue()))
             .toList();
     List<TableMetadata> list = new ArrayList<>();
-    list.addAll(schema.getTablesIncludingExternal());
```

Comment:
why would getTablesIncludingExternal fetch non related tables ?

Reply:
getTablesIncludingExternal is no longer called after this change and should be removed, sonar should have found this ( maybe it did ? )

Reply:
will double check and delete if not used